pkg/virt-handler/vm_test.go

```diff
@@ -1672,7 +1672,8 @@ var _ = Describe("VirtualMachineInstance", func() {
 			virtClient.CoreV1().PersistentVolumeClaims(vmi.Namespace).Create(testBlockPvc)
 			blockMigrate, err := controller.checkVolumesForMigration(vmi)
 			Expect(blockMigrate).To(BeTrue())
-			Expect(err).To(Equal(fmt.Errorf("cannot migrate VMI with non-shared PVCs")))
+			Expect(err).ToNot(BeNil())
+			Expect(err.Error()).To(ContainSubstring("cannot migrate VMI: PVC "))
```

Comment:
In this case it should be super trivial to match the whole error message exactly, could you do that instead?

Reply:
yes